doc/getting_started/installation_win.rst

```diff
@@ -87,6 +87,39 @@ packages from their respective websites.
       cd %userprofile%\zephyr
       pip3 install --user -r scripts/requirements.txt
 
+#. If you want to be able to run the ``menuconfig`` interactive configuration
```

Comment:
#. If you'll be using the ``menuconfig`` interactive Zephyr configuration interface
       you'll need to download a python curses library appropriate for your Windows
       Python version (32-bit or 64-bit).  (**Note: Do not assume that you
       are running a 64-bit version of Python because your OS is 64-bit.**)

       Run the command::
    
          python -VV

       and you'll see output something like this::

          Python 3.6.5 (v3.6.5:f59c0932b4, Mar 28 2018, 16:07:46) [MSC v.1900 32 bit (Intel)]
     
       (Note this is saying we're using a ``32 bit`` version of Python.)

       Next, visit the `Python Curses libraries web site
       <https://www.lfd.uci.edu/~gohlke/pythonlibs/#curses>`_ and download
       the latest ``curses‑2.2+utf8`` wheel (:file:`.whl` file) for your Python
       version (32- or 64-bit). 

       Install the library with the following command (matching the version you downloaded)::

          pip3 install --user "~/Downloads/curses-2.2+utf8-cp37-cp37m-win32.whl"

       .. note::  We recomment using the ``utf8`` version of the wheel to properly handle
          Unicode character handling.

Reply:
What do you think if this version?

Looks like we might be splitting up the pull request and putting in the documentation separately, but I'll get back to it then.

```rst
If you want to use the Zephyr ``menuconfig`` configuration interface, you will
need a Python curses module for Windows. The following instructions will
install one.

First, run this command to check which Python version you are using:

.. code-block:: console

   python -VV

You will get output like the following:

.. code-block:: console

   Python 3.6.5 (v3.6.5:f59c0932b4, Mar 28 2018, 16:07:46) [MSC v.1900 32 bit (Intel)]

In this case, the output tells us that we are using a 32-bit version of
Python 3.6.

  ... note::

      Do not assume that Python must be 64-bit if you are running a 64-bit
      OS. What matters here is whether the Python installation is 32- or
      64-bit.

Next, go to `this web page
<https://www.lfd.uci.edu/~gohlke/pythonlibs/#curses>`_ and download the
``curses-*+utf8`` wheel (:file:`.whl` file) that matches your Python version
and bit width.

Install the wheel with the following command, giving the path you downloaded
it to:

.. code-block:: console

   pip3 install --user "Downloads\curses-2.2+utf8-cp36-cp36m-win32.whl"

The :file:`.whl` file used in this example would be appropriate for the
``python -VV`` output above.

  .. note::

     The non-``+utf8`` versions of the wheels are likely to work as well. The
     ``+utf8`` versions might avoid Unicode issues in obscure cases.
```

`~` doesn't work well with e.g. `cmd.exe`, so that's why I avoided it.